src/main/java/com/mccneb/edu/demo/model/User.java

```diff
@@ -0,0 +1,43 @@
+package com.mccneb.edu.demo.model;
+
+import jakarta.persistence.*;
```

Comment:
Industry standard doesn't like `*` imports, we can change this in your IDE settings.